.github/scripts/process_commit.py

```diff
@@ -71,9 +71,13 @@ def get_repo_labels() -> List[str]:
 
 def post_pytorch_comment(pr_number: int, merger: str) -> Any:
     message = {'body' : f"Hey {merger}. \
-    You merged this PR, but no release notes category and topic labels were added. \
-    The list of valid release and topic labels is available \
-    https://github.com/pytorch/pytorch/labels?q=release+notes+or+topic"}
+    You merged this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
```

Comment:
Nit: most of PRs are not merged, but committed
```suggestion
    You've committed  this PR, but no it does not have both a 'release notes: ...' and 'topics: ...' label. \
```